typegate/src/typegate/artifacts/shared.ts

```diff
@@ -0,0 +1,169 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { createHash } from "node:crypto";
+import { S3, S3ClientConfig } from "aws-sdk/client-s3";
+import {
+  deinitUploadUrlStore,
+  getLocalPath,
+  initUploadUrlStore,
+  STORE_DIR,
+  STORE_TEMP_DIR,
+  UploadUrlStore,
+} from "./local.ts";
```

Comment:
I suggest to put shared types and constants in `artifacts/mod.ts` or somewhere else...

Reply:
Done.